libs/ui-watt/src/lib/components/button/watt-button.stories.ts

```diff
@@ -63,3 +69,24 @@ export const Button = ButtonTemplate.bind({});
 Button.args = {
   type: 'text',
 };
+
+const IconButtonTemplate: Story<WattIconButtonComponent> = (args) => ({
+  props: args,
+  template: `<watt-icon-button icon="${args.icon}" [disabled]="${args.disabled}"></watt-icon-button>`,
```

Comment:
This template doesn't seem to be reflected in the docs:
![image](https://user-images.githubusercontent.com/86852536/141104847-dc9e108f-f92d-40d2-b7ee-f8f1477ad82a.png)

Reply:
Fixed.